homeassistant/components/hive/config_flow.py

```diff
@@ -103,6 +103,7 @@ async def async_setup_hive_entry(self):
 
         # Setup the config entry
         self.data["tokens"] = self.tokens
+        self.data["device_data"] = await self.hive_auth.getDeviceData()
```

Comment:
The config flow tests fail because this is returning a tuple `(None, None, None)`